src/locale/nb.js

```diff
@@ -2,54 +2,54 @@
 //! locale : Norwegian Bokmål [nb]
 //! authors : Espen Hovlandsdal : https://github.com/rexxars
 //!           Sigurd Gartmann : https://github.com/sigurdga
+//!           Ronny Hanssen : https://github.com/spiralis
 
 import moment from '../moment';
 
 export default moment.defineLocale('nb', {
-    months : 'januar_februar_mars_april_mai_juni_juli_august_september_oktober_november_desember'.split('_'),
-    monthsShort : 'jan._feb._mars_april_mai_juni_juli_aug._sep._okt._nov._des.'.split('_'),
-    monthsParseExact : true,
-    weekdays : 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
-    weekdaysShort : 'sø._ma._ti._on._to._fr._lø.'.split('_'),
-    weekdaysMin : 'sø_ma_ti_on_to_fr_lø'.split('_'),
-    weekdaysParseExact : true,
-    longDateFormat : {
-        LT : 'HH:mm',
-        LTS : 'HH:mm:ss',
-        L : 'DD.MM.YYYY',
-        LL : 'D. MMMM YYYY',
-        LLL : 'D. MMMM YYYY [kl.] HH:mm',
-        LLLL : 'dddd D. MMMM YYYY [kl.] HH:mm'
+    months: 'januar_februar_mars_april_mai_juni_juli_august_september_oktober_november_desember'.split('_'),
+    monthsShort: 'jan_feb_mar_apr_mai_jun_jul_aug_sep_okt_nov_des'.split('_'),
```

Comment:
`monthsShort` should be `'jan._feb._mars_apr._mai_juni_juli_aug._sep._okt._nov._des.'.split('_')` per [the Language Council of Norway](https://www.sprakradet.no/sprakhjelp/Skriveregler/Forkortinger/). Months with 4 or fewer letters are not abbreviated, while others are exactly 3 letters with a dot at the end.

Reply:
I see that this is the official  ruling. As I mentioned above. 

The typical usecase is as table headers, and for those the dots seem very misplaced and unnecessary. 

In a sentence the dot-form makes more sense. In headers not so much. In prosa I don't think I have "ever" seen abbreviated forms though, neither of days nor months. Except maybe only in SMS messages, where I am pretty sure that they were always without dots. 

But in tables the abbreviated forms are frequent, and dots are really not needed there.